### PR TITLE
[sdk] Refactor sdk/src/api folder structures and control exports

### PR DESCRIPTION
**Pull request by haVincy**

### CODE REVIEW

**Comment by haVincy** [comment]
Trigger the dev branch CI check here

- https://github.com/LIQNFT/candy-shop/actions/runs/2519558476

File: core/ui/src/constant/Orders.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { OrderSortBy } from '@liqnft/candy-shop-sdk';
+import { OrderSortBy } from '@liqnft/candy-shop-types';
```
Comment (haVincy) [inline core/ui/src/constant/Orders.ts:1]: Minor refactor here in core/ui to put `OrderSortBy` into the types.

**Review by RustySol — APPROVED** [review]

**Review by nibbus0x — APPROVED** [review]
Changes look good to me! Seems like a straight forward refactor

**Comment by haVincy** [comment]
Thx guys, merging.

**Pull request merged**